app/assets/javascripts/curation_concerns/file_manager/sorting.es6

```diff
@@ -82,4 +87,40 @@ export default class SortManager {
       }
     ).toArray()
   }
+
+  get alpha_sort_button() {
+    return $("*[data-action='alpha-sort-action']")
+  }
+
+  initialize_alpha_sort_button() {
+    Blacklight.onLoad(() => {
```

Comment:
I don't think the sorting manager gets triggered until Blacklight's loaded, so this shouldn't be necessary.